relay-config/src/config.rs

```diff
@@ -521,6 +513,46 @@ impl Default for Metrics {
             hostname_tag: None,
             buffering: true,
             sample_rate: 1.0,
+        }
+    }
+}
+
+/// Controls processing of metrics and metric metadata.
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(default)]
+struct SentryMetrics {
```

Comment:
`SentryMetrics` seems ok.  But what about `DeveloperMetrics` or `SentryDeveloperMertrics`?

Reply:
I feel like `Developer` implies DDM/custom metrics but this is supposed to be for config for all metrics.